tests/baselines/reference/augmentedTypesClass2a.errors.txt

```diff
@@ -1,15 +1,23 @@
 tests/cases/compiler/augmentedTypesClass2a.ts(2,7): error TS2300: Duplicate identifier 'c2'.
+tests/cases/compiler/augmentedTypesClass2a.ts(2,7): error TS2300: Duplicate identifier 'c2'.
+tests/cases/compiler/augmentedTypesClass2a.ts(3,10): error TS2300: Duplicate identifier 'c2'.
 tests/cases/compiler/augmentedTypesClass2a.ts(3,10): error TS2300: Duplicate identifier 'c2'.
 tests/cases/compiler/augmentedTypesClass2a.ts(4,5): error TS2300: Duplicate identifier 'c2'.
 
 
-==== tests/cases/compiler/augmentedTypesClass2a.ts (3 errors) ====
+==== tests/cases/compiler/augmentedTypesClass2a.ts (5 errors) ====
     //// class then function
     class c2 { public foo() { } } // error
           ~~
+!!! error TS2300: Duplicate identifier 'c2'.

```

Comment:
duplicate duplicate identifier error here

Reply:
It's technically not duplicate since it adds a related span, otherwise it wouldn't display (it'd get deduped). We're issuing the duplicate errors in two places, once in the binder, and once in the checker. The binder is triggered by the `var c2` later in the file (that's a symbol exclude flags violation, so it isn't even added to the symbol), then we get into the checker, look into the specifics of the class/function merge, see that the class is non-ambient, and _also_ issue an error (arguably a better one, with related spans, but only containing those declarations which actually made it into the symbol).

Reply:
After talking in person, improving the checker-created error message as in the above TODO will make this particular test less confusing.